src/layout.rs

```diff
@@ -474,7 +499,8 @@ impl Layout {
     ///
     /// This method stores the result of the computation in a thread-local cache keyed on the layout
     /// and area, so that subsequent calls with the same parameters are faster. The cache is a
-    /// simple HashMap, and grows indefinitely (<https://github.com/ratatui-org/ratatui/issues/402>).
+    /// LruCache, and grows until `DEFAULT_CACHE_SIZE` is reached by default, if the cache is
```

Comment:
```suggestion
    /// LruCache, and grows until [`Self::DEFAULT_CACHE_SIZE`] is reached by default, if the cache is
```

Reply:
Hi @joshka 

Thanks again for your time. I will improve the documentation as suggested.

Reply:
You're welcome. Thanks for the changes. This looks good now. I'll recommend the other maintainers take a second review, but generally LGTM.